config/application.rb

```diff
@@ -4,9 +4,10 @@
 
 # Select only parts we need from rails/all
 require 'rails'
-require 'active_record/railtie'
 require 'action_controller/railtie'
 require 'active_job/railtie'
+require 'active_record/railtie'
+require 'action_cable/engine'
```

Comment:
This was require to avoid https://github.com/sul-dlss/preservation_catalog/pull/2360/files#diff-053aa3ea46cb7c6a649e0d0fc592ce729b66e1d44be475516fd58857ba695862L6 raising an error in CI (does not get raised when running the tests locally).